### DISCUSSION TITLE
Projects (Beta) Suggestion: Add hill charts

### DISCUSSION THREAD

**Discussion by rayalan** [community/community#13119] [Projects and Issues]
I'm really enjoying the projects beta. In order to easily present an upward visibility to management and project tracking, I'd love to see integrated hill charts (think hillia.app). The goal here is then to able to see project progress, not just the progress of individual items. We've found that Hill Charts work really well for us in two ways: 

* They indicate the overall confidence/certainty in where the project is at. How many more unknowns are expected? How many more tasks remain to be done? And they answer these questions without requiring creating task entries ahead of time, allowing the actual tasks to nicely shift.
* They make it easier to present status in a way that management tends to think of it - I would feature Foo to ship - not in the per-task way that developers or teams breakdown the work. 

With projects, I'd probably suggest something like this: 

* Each project line corresponds to a Hill Chart task. 
* Each project line is mapped to a Hill Chart scope via label.
* Each milestone has a Hill Chart, which is associated with certain labels. 
* There is a view of the hill chart for each milestone.
* There is a view of multiple hill charts.
* * Add the ability to indicate how far along a hill chart a particular label is. 
* There can be additional hill-chart specific editing views, such as adding tasks which automatically get the corresponding label to be in the right scope/hill chart. 

This creates the ability to view hill charts by management, makes it easy for developers to associate an issue with a Hill Chart, and lets management create different Hill Charts by associating different labels with milestones. It avoids creating too many new capabilities just for Hill Charts, but rather leverages the existing capabilities with a distinct view.

**Comment by HonzaMikula** [comment]
I would love to see hill chart as well. I think this chapter of the Shape Up book nicely describe the reasoning behind hill chart. [https://basecamp.com/shapeup/3.4-chapter-13](https://basecamp.com/shapeup/3.4-chapter-13)
